### PR TITLE
Update wording of 01-Getting-Started

### PR DESCRIPTION
**Pull request by imidom**
Hi! I'm learning Javascript and have found these tutorials to be incredibly helpful. While the articles are very clear, some of the wording isn't natural to me as an English speaker. I was hoping I could contribute to this project by updating the wording of the articles to read more like natural English. If you like these edits, let me know and I'll make my way through the rest of the articles as I read them. Thanks for creating this.

### CODE REVIEW

**Comment by iliakan** [comment]
Thank you!

File: 1-js/01-getting-started/1-intro/article.md [hunk 1]
```diff
@@ -6,29 +6,29 @@ Let's see what's so special about JavaScript, what we can achieve with it, and w
...
-The programs in this language are called *scripts*. They can be written right in the HTML and executed automatically as the page loads.
+The programs in this language are called *scripts*. They can be written right in a web page's HTML and executed automatically as the page loads.
 
-Scripts are provided and executed as a plain text. They don't need a special preparation or a compilation to run.
+Scripts are provided and executed as plain text. They don't need special preparation or compilation to run.
 
 In this aspect, JavaScript is very different from another language called [Java](https://en.wikipedia.org/wiki/Java_(programming_language)).
 
 ```smart header="Why <u>Java</u>Script?"
-When JavaScript was created, it initially had another name: "LiveScript". But Java language was very popular at that time, so it was decided that positioning a new language as a "younger brother" of Java would help.
```
[the remark this answers is not in the document]
Comment (iliakan) [reply 1-js/01-getting-started/1-intro/article.md]: Please leave it as is or add "programming language".

File: 1-js/01-getting-started/1-intro/article.md [hunk 2]
```diff
@@ -86,14 +86,13 @@ There are at least *three* great things about JavaScript:
 ```compare
 + Full integration with HTML/CSS.
 + Simple things are done simply.
-+ Supported by all major browsers and enabled by default.
++ Support by all major browsers and enabled by default.
```
Comment (iliakan) [inline 1-js/01-getting-started/1-intro/article.md]: Does not look good.

File: 1-js/01-getting-started/1-intro/article.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (imidom) [reply 1-js/01-getting-started/1-intro/article.md]: OK! We just need to drop a "The" in there-->
```suggestion
When JavaScript was created, it initially had another name: "LiveScript". But the Java programming language was very popular at that time, so it was decided that positioning a new language as a "younger brother" of Java would help.
```

File: 1-js/01-getting-started/1-intro/article.md (hunk 2 above)
Comment (imidom) [reply 1-js/01-getting-started/1-intro/article.md]: I'm sorry for this long explanation:

The reason I made this change was to help the last line match the structure of the first one. The [changing structure](https://www.iup.edu/writingcenter/writing-resources/grammar/parallel-structure/) of these bullets makes them a little bit harder to follow.

If each bullet starts with a verb, they'll flow more naturally together. A quick test would be to put "Javascript is..." in front of each of these to see if they form a well-structured sentence: 
  + [Javascript is] Fully integrated with HTML/CSS.
  + [Javascript is] Supported by all major browsers and enabled by default.
  + [Javascript is] Able to do simple things in simple ways.

Let me know if you would like me to make these changes or keep as is.

File: 1-js/01-getting-started/1-intro/article.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (imidom) [reply 1-js/01-getting-started/1-intro/article.md]: Sounds good--we'll just need to add a "the" to keep "language" or "programming language" in there.
```suggestion
When JavaScript was created, it initially had another name: "LiveScript". But the Java programming language was very popular at that time, so it was decided that positioning a new language as a "younger brother" of Java would help.
```

**Comment by iliakan** [comment]
You're a native speaker, is that correct?

**Comment by imidom** [comment]
I am, but I do not want to make changes you don't agree with. This is a great tutorial--I want to help the wording flow more naturally in English without changing the content.

**Comment by iliakan** [comment]
Okay, then I assume you're sure of the new wording ;)

**Comment by imidom** [comment]
Sounds good. I'll try to submit a pull request for a chapter a day. Thanks again for creating this tutorial.

**Pull request merged**